tasks/client_config.yaml

```diff
@@ -1,12 +1,24 @@
 ---
+- name: Check for root password
+  stat:
+    path: /usr/local/etc/mysql-password
+  register: mysql_password
+
+- name: create root password
+  command: "pwgen 25 1"
```

Comment:
Can we use Ansible's `password` lookup for this?
https://docs.ansible.com/ansible/latest/plugins/lookup/password.html

Then we can also remove the `pwgen` dependency.

Reply:
Yes we can.

Reply:
Fixed with 347cb9ac5dc5cb1a677646e0156dff326492becc